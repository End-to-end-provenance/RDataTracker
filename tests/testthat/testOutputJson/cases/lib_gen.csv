"package","version"
"base","3.5.1"
"datasets","3.5.1"
"ggplot2","2.2.1"
"graphics","3.5.1"
"grDevices","3.5.1"
"methods","3.5.1"
"RDataTracker","2.27.0"
"stats","3.5.1"
"utils","3.5.1"
