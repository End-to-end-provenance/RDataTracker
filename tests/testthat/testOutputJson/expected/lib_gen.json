
		// library nodes - prov collections
		"rdt:l1": {
			"name": "base",
			"version": "3.5.1",
			"prov:type": {
				"$": "prov:Collection",
				"type": "xsd:QName"
			}
		},
		"rdt:l2": {
			"name": "datasets",
			"version": "3.5.1",
			"prov:type": {
				"$": "prov:Collection",
				"type": "xsd:QName"
			}
		},
		"rdt:l3": {
			"name": "ggplot2",
			"version": "2.2.1",
			"prov:type": {
				"$": "prov:Collection",
				"type": "xsd:QName"
			}
		},
		"rdt:l4": {
			"name": "graphics",
			"version": "3.5.1",
			"prov:type": {
				"$": "prov:Collection",
				"type": "xsd:QName"
			}
		},
		"rdt:l5": {
			"name": "grDevices",
			"version": "3.5.1",
			"prov:type": {
				"$": "prov:Collection",
				"type": "xsd:QName"
			}
		},
		"rdt:l6": {
			"name": "methods",
			"version": "3.5.1",
			"prov:type": {
				"$": "prov:Collection",
				"type": "xsd:QName"
			}
		},
		"rdt:l7": {
			"name": "RDataTracker",
			"version": "2.27.0",
			"prov:type": {
				"$": "prov:Collection",
				"type": "xsd:QName"
			}
		},
		"rdt:l8": {
			"name": "stats",
			"version": "3.5.1",
			"prov:type": {
				"$": "prov:Collection",
				"type": "xsd:QName"
			}
		},
		"rdt:l9": {
			"name": "utils",
			"version": "3.5.1",
			"prov:type": {
				"$": "prov:Collection",
				"type": "xsd:QName"
			}
		}
